src/pages/instances/InstanceDetailHeader.tsx

```diff
@@ -39,8 +36,9 @@ const InstanceDetailHeader: FC<Props> = ({
   const navigate = useNavigate();
   const toastNotify = useToastNotification();
   const controllerState = useState<AbortController | null>(null);
-  const { data: settings } = useSettings();
-  const isClustered = isClusteredServer(settings);
+  const [isLargeScreen, setIsLargeScreen] = useState(isWidthBelow(1200));
+
+  useEventListener("resize", () => setIsLargeScreen(isWidthBelow(1200)));
```

Comment:
I think this can move into the `InstanceDetailActions`, we don't need it here, right?